hil-test/tests/qspi.rs

```diff
@@ -206,7 +205,9 @@ mod tests {
             }
         }
 
-        let dma_channel = dma_channel.configure(false, DmaPriority::Priority0);
+        let dma_channel = dma_channel
+            .configure(false, DmaPriority::Priority0)
+            .degrade();
```

Comment:
Is this explicit `.degrade()` intentional?

Reply:
No, it was supposed to be cleaned up, thanks for the notice.